This patch is applied on top of the other keyboard patches and is dependant
on them.

Also, swapping your capslock and ctrl keys is something that your Operating
System can do, so patching the EC firmware is quite an overkill for this
task.

--- t430.G1HT35WW.img.prepatched.hex	2019-01-15 03:40:49.758801095 +0000
+++ t430.G1HT35WW.img.swapctrlcaps.hex	2019-01-15 03:41:10.240081095 +0000
@@ -8505,7 +8505,7 @@
 00021380  00 00 00 00 b2 00 00 00  7c 11 02 00 40 00 00 00  |........|...@...|
 00021390  20 13 02 00 e0 12 02 00  0b 00 00 00 60 13 02 00  | ...........`...|
 000213a0  01 02 11 10 1f 6e 2e 83  00 00 00 00 00 00 00 00  |.....n..........|
-000213b0  70 03 12 1e 20 2d 2f 00  00 00 00 00 00 00 00 00  |p... -/.........|
+000213b0  70 03 12 3a 20 2d 2f 00  00 00 00 00 00 00 00 00  |p..: -/.........|
 000213c0  71 04 13 72 21 73 30 00  00 00 00 00 00 00 00 00  |q..r!s0.........|
 000213d0  06 05 14 15 22 23 31 32  00 00 00 00 00 00 00 00  |...."#12........|
 000213e0  07 08 17 16 25 24 34 33  00 00 00 00 00 00 00 00  |....%$43........|
@@ -8519,8 +8519,8 @@
 00021460  50 51 00 00 00 53 7e 4f  00 00 00 00 00 00 00 00  |PQ...S~O........|
 00021470  00 7c 7d 00 00 3c 00 3e  00 00 00 00 00 00 00 00  |.|}..<.>........|
 00021480  00 00 00 2c 00 00 39 00  00 00 00 00 00 00 00 00  |...,..9.........|
-00021490  3a 00 00 00 00 00 40 00  00 00 00 00 00 00 00 96  |:.....@.........|
-000214a0  00 9d 00 9e 9f 4a 3a 9c  7d 7e 00 00 00 00 00 00  |.....J:.}~......|
+00021490  1e 00 00 00 00 00 40 00  00 00 00 00 00 00 00 96  |......@.........|
+000214a0  00 9d 00 9e 9f 4a 1e 9c  7d 7e 00 00 00 00 00 00  |.....J..}~......|
 000214b0  ff 00 7f 00 7f 00 ff 00  ff 00 ff 00 ff 00 ff 00  |................|
 000214c0  ff 00 8b 00 ff 00 d3 00  e3 00 a6 00 48 00 41 80  |............H.A.|
 000214d0  fa 03 00 00 a0 97 00 00  10 01 00 00 a0 13 02 00  |................|
